vnext/ReactUWP/Views/SwitchViewManager.cpp

```diff
@@ -160,5 +160,17 @@ bool SwitchViewManager::UpdateProperty(
   return true;
 }
 
+void SwitchViewManager::DispatchCommand(
+    XamlView viewToUpdate,
+    const std::string &commandId,
+    const folly::dynamic &commandArgs) {
+  if (commandId == "setValue") {
+    auto value = commandArgs[0].asBool();
+    viewToUpdate.as<winrt::ToggleSwitch>().IsEnabled(value);
```

Comment:
This command is kind of weird. It's iOS only upstream, and in their impl doesn't touch native props either. From what I can tell it's to fix an iOS quirk, but implementing it doesn't seem to cause us any issues, and let's us fully share iOS JS code.